src/insns/load_16bit.adoc

```diff
@@ -50,10 +38,10 @@ Standard load instructions, authorized by the capability in <<ddc>>.
 include::load_exceptions.adoc[]
 
 Prerequisites for {cheri_cap_mode_name} C.LD::
-RV64, and {c_cheri_base_ext_names}
+RV64 or RV32 and Zclsd, and {c_cheri_base_ext_names}
```

Comment:
```suggestion
RV64 or RV32 with Zclsd, and {c_cheri_base_ext_names}
```

I wonder if this would be less ambiguous? Right now it can also be read as `(RV64 || RV32) && Zclsd && Zcheripurecap`